src/Interpreters/Context.h

```diff
@@ -14,6 +14,7 @@
 #include <Common/MultiVersion.h>
 #include <Common/OpenTelemetryTraceContext.h>
 #include <Common/RemoteHostFilter.h>
+#include <Common/ThreadPool.h>
```

Comment:
This include looks redundant

Reply:
`ThreadPool` is used in this header

Reply:
I found only two usages: two functions return `ThreadPool &`, but it does not require the definition